src/app/cube/cube.module.ts

```diff
@@ -20,9 +20,9 @@ import { SharedModule } from '../shared/shared.module';
 import { CubeComponent } from './cube.component';
 import { BrowseComponent } from './browse/browse.component';
 import { UserProfileComponent } from './user-profile/user-profile.component';
-import { MatTabsModule } from '@angular/material/tabs';
+import { MatLegacyTabsModule as MatTabsModule } from '@angular/material/legacy-tabs';
```

Comment:
This legacy documenaton pls

Reply:
https://material.angular.io/guide/mdc-migration